docs/reference/requirement-sets/identity-api-requirement-sets.md

```diff
@@ -14,9 +14,12 @@ Office Add-ins run across multiple versions of Office. The following table lists
 
 |  Requirement set  | Office 2013 or later on Windows<br>(one-time purchase) | Office on Windows<br>(connected to a Microsoft 365 subscription) |  Office on iPad<br>(connected to a Microsoft 365 subscription)  |  Office on Mac<br>(connected to a Microsoft 365 subscription)  | Office on the web  |
 |:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-| IdentityAPI 1.3  | N/A | 2008 (build 13127.20000) or later | Coming soon | 16.40 or later | August, 2020* |
+| IdentityAPI 1.3  | N/A | 2008 (build 13127.20000) or later | Coming soon | 16.40 or later | Microsoft SharePoint Online and OneDrive\* |
```

Comment:
Should the date be kept since you explain in the note about the restrictions?

Reply:
I think we were trying to avoid putting in a specific date, and just keep it to the note.